qc_tests/streaks.py

```diff
@@ -203,29 +532,50 @@ def rsc(station, var_list, config_dict, full=False, plots=False, diagnostics=Fal
     for var in var_list:
 
         obs_var = getattr(station, var)
+        if obs_var.name == "wind_direction":
+            wind_speed = copy.deepcopy(getattr(station, "wind_speed"))
+        else:
+            wind_speed = None
 
         # need to have at least two observations to get a streak
         if len(obs_var.data.compressed()) >= 2:
-
-            # repeating (straight) strings
             if full:
```

Comment:
What is 'full'? What does it mean?

Reply:
I've remembered why "full" - for "full reprocessing"